etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -23,6 +23,9 @@ spec:
     metadata:
       annotations:
         checksum/storage-secret: {{ include (print $.Template.BasePath "/pachd/storage-secret.yaml") . | sha256sum }}
+          {{- if .Values.pachd.annotations -}}
+            {{ toYaml .Values.pachd.annotations | nindent 4 }}
```

Comment:
is this guy over indented?

Reply:
it doesn't matter for the render but I cleaned it up